projects/lib/src/lib/models/node-transform.ts

```diff
@@ -0,0 +1,16 @@
+import { LuigiNode } from './luigi';
+
+export interface ValidationMessage {
+  message: string;
+}
+
+export interface ValidationResult {
+  parsedConfiguration?: string;
+  validationErrors?: ValidationMessage[];
```

Comment:
are the ValidationMessage always errors? if so I will rename to
```suggestion
  validationErrors?: ValidationError[];
```

Reply:
or perhaps rename validationErrors to validationMessages

Reply:
this is good  validationErrors?: ValidationError[];